core/templates/components/forms/custom-forms-directives/html-select.component.css

```
@@ -0,0 +1,79 @@
+  margin-bottom: 0;
+}
+
+.oppia-html-select .oppia-dropdown-menu {
+  max-width: 80%;
+  overflow: auto;
+  width: auto;
+}
+
+.oppia-html-select .oppia-dropdown-menu-text {
+  white-space: normal;
+  word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)